modules/instrument_builder/jsx/react.instrument_builder.js

```diff
@@ -65,43 +65,47 @@ class LoadPane extends Component {
   }
   // Render the HTML
   render() {
-    let alert = '';
+    let alert = {
+      message: '',
+      details: '',
+      display: 'none',
+    };
     // Set up declared alerts, if there is any.
     switch (this.state.alert) {
       case 'success':
-        alert = (
-          <div className='alert alert-success alert-dismissible' role='alert'>
-            <button type='button' className='close' onClick={this.resetAlert}><span aria-hidden='true'>&times;</span></button>
-            <strong>Success!</strong> Instrument Loaded
-          </div>
-        );
+        alert = {
+          message: 'Success!',
+          details: 'Instrument Loaded',
+          display: 'block',
+        };
         break;
       case 'typeError':
-        alert = (
-          <div className='alert alert-danger alert-dismissible' role='alert'>
-            <button type='button' className='close' onClick={this.resetAlert}><span aria-hidden='true'>&times;</span></button>
-            <strong>Error!</strong> Wrong file format
-          </div>
-        );
+        alert = {
+          message: 'Error!',
+          details: 'Wrong file format',
+          display: 'block',
+        };
         break;
       case 'duplicateEntry':
-        alert = (
-          <div className='alert alert-danger alert-dismissible' role='alert'>
-            <button type='button' className='close' onClick={this.resetAlert}>
-              <span aria-hidden='true'>&times;</span>
-            </button>
-            <strong>Error!</strong><br/>
-              {this.state.alertMessage}
-          </div>
-        );
+        alert = {
+          message: 'Error!',
+          details: this.state.alertMessage,
+          display: 'block',
+        };
         break;
       default:
         break;
     }
     return (
       <TabPane Title='Load Instrument' {...this.props}>
         <div className='col-sm-6 col-xs-12'>
-          {alert}
+          <div id='load_alert' style={{display: alert.display}} className='alert alert-danger alert-dismissible' role='alert'>
```

Comment:
alert-danger is set even when the alert is successful?

Reply:
Nice catch @zaliqarosli 🙂I added a fix so it will show alert-success when success.